tests/integration/test_clip.py

```diff
@@ -172,6 +184,18 @@ def test_out(ndim):
     assert np.array_equal(out_np, out_num)
 
 
+def test_out_with_array_amin():
+    array = ((2, 3, 4), (3, 4, 5), (6, 6, 12))
+    amin = (2, 3, 1)
+    amax = 6
+    np_arr = np.array(array)
+    num_arr = num.array(array)
+    out_np = out_num = np.zeros(np_arr.shape)
```

Comment:
same comment as above

Reply:
Definitely my mistake, and `out_num` in this one is not even meant to be a np array. Fixed.